src/ui/page/share/dataset/steps/summary/summary.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-lines-per-function */
+import { type FC, useState, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useFileStore, useAppStore } from '@/ui/store'
+import { type Item, List } from '@/ui/component/list/list'
+import { Icon } from '@/ui/component/icon/icon'
+import { KnowFee } from '@/ui/view/share/knowFee/knowFee'
+import { Checkbox } from '@/ui/component/checkbox/checkbox'
+import './summary.scss'
+
+export const Summary: FC = () => {
+  const { t } = useTranslation('share')
+  const { files } = useFileStore(state => ({
+    files: state.filesDescriptor
+  }))
+
+  const certifyText = t('share:share.dataset.summaryCertify')
+
+  const [isChecked, setChecked] = useState<boolean>(false)
+  const handleCheckedChange = useCallback(() => setChecked(s => !s), [])
+
+  const items: Item[] = files()().map(({ id, name }) => ({
+    content: <p>{name}</p>,
+    id,
+    key: id,
+    leftElement: (
+      <div>
+        <Icon name="folder-outlined" />
+      </div>
+    )
+  }))
+
+  const formItems = useAppStore(state => state.shareData.form)
+  const filteredformItems = formItems.filter(item => item.label !== 'fee')
+
+  return (
+    <div className="okp4-dataverse-portal-share-dataset-summary-container">
+      <h2>{t('share:share.dataset.summary')}</h2>
+      <div className="okp4-dataverse-portal-share-dataset-summary-left">
+        <div className="okp4-dataverse-portal-share-dataset-summary-text-container">
+          {filteredformItems.map(({ label, value }, index) => (
+            <div className="okp4-dataverse-portal-share-dataset-summary-text" key={index}>
+              <h3>{label.replace(/([A-Z])/g, ' $1')}</h3>
+              <span>{value.toString()}</span>
```

Comment:
`<Tag />` should be used here I guess?

![Capture d’écran 2023-07-20 à 15 18 15](https://github.com/okp4/dataverse-portal/assets/44092179/837839f4-63e3-4780-8281-7c7d9a36877f)

Reply:
`value.toString()` is a potential bug since `value` answers to this type `type FormItemValue = string | string[] | number | DatePickerValue`

Reply:
Hmm I wasn't sure how else to deal with it because it was throwing me an error that `DatePickerValue` is not a `ReactNode`. Any suggestions?

Reply:
@oussama-aitbaziz-dev good catch! Tags has been added in [606fb67](https://github.com/okp4/dataverse-portal/pull/404/commits/606fb670dfb867917798584fcc25b8376ef85ee6).